content/guides/models/track/log/distributed-training.md

```diff
@@ -149,9 +246,9 @@ if __name__ == "__main__":
     main()
 ```
 
-#### Share a W&B Run
+### Share a run
 
-Pass a W&B Run object as an argument to share W&B Runs between processes:
+Pass a W&B run object as an argument to share runs between processes:
```

Comment:
You capitalize Run up in line 228 but not here. This doesn't feel to me like a use case exactly.